src/main/resources/fabric.mod.json

```diff
@@ -1,27 +1,29 @@
 {
```

Comment:
You removed the artists attribution by excluding the contributors block. It is also missing the waila entry point data (https://github.com/TehNut/HWYLA/blob/1.14_fabric/README.md#information-for-developers)

Reply:
Fixed contributor block and added waila block according to docs.